### PR TITLE
Allow for 1 more month when registering LMP

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
Ready for review @DevChima @bruskiza @rudigiesler @KaitCrawford if you have a moment.

File: src/utils_project.js [hunk 1]
```diff
@@ -157,7 +157,9 @@ go.utils_project = {
     is_valid_month: function(today, choiceYear, choiceMonth, monthsValid) {
         // function used to validate months for states 5A/5B & 12A/12B
 
-        var choiceDate = new moment(choiceYear+choiceMonth, "YYYYMM");
+        // NOTE: make sure we cast numbers to strings instead
+        //       of adding up numbers
+        var choiceDate = new moment('' + choiceYear + choiceMonth, "YYYYMM");
```
Comment (smn) [inline src/utils_project.js:162]: Not sure if this is necessary but it seems too easy to insert integers like `2017` and `1` and have that get parsed as `2018` or something.

File: test/utils_project.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                go.utils_project.is_valid_month(today, 2016, 1, 11),
+                true);
+        });
+
+        it('should pass for dec last year', function () {
+            var today = new moment('2016-09-01', 'YYYY-MM-DD');
+            assert.equal(
+                go.utils_project.is_valid_month(today, 2015, 12, 11),
+                true);
+        });
```
Comment (smn) [inline test/utils_project.test.js:19]: These were the dates that were reported to not be working.

**Comment by DevChima** [comment]
👍

**Pull request merged**